package.json

```diff
@@ -2,7 +2,7 @@
   "name": "marked",
   "description": "A markdown parser built for speed",
   "author": "Christopher Jeffrey",
-  "version": "0.3.6",
+  "version": "0.3.7",
```

Comment:
Thanks for the PR! Could you revert this? We're not publishing our forked package, so I don't think we need to update this